app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+  <%= link_to "sample app", "#", id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("Home"), "#" %></li>
```

Comment:
`Home` tên beiens ko bắt đầu với từ cái viết hoa, em self chekc toàn bộ các i18n nehs